makejdk.sh

```diff
@@ -260,12 +260,9 @@ buildAndTestOpenJDKViaDocker()
 
 testOpenJDKInNativeEnvironmentIfExpected()
 {
-  if [[ ! -z $JTREG ]]; then
-    if [[ ! -z $JTREG_TEST_SUBSETS ]]; then
+  if [[ "${JTREG}" == "true" ]];
+  then
```

Comment:
@karianna should work the build jobs above (see links) dont set `JTREG_TEST_SUBSETS`, the below line
`$WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JTREG_TEST_SUBSETS` becomes `$WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME`